tools/ts-node/ts_node.test.ts

```diff
@@ -0,0 +1,11 @@
+import { makeToolTestConfig, toolTest } from "tests";
+toolTest({
```

Comment:
use `toolInstallTest` instead (requires fewer args and uses the built-in `health_checks`)

Reply:
done